src/osemgrep/configuring/Engine_type.ml

```diff
@@ -1,3 +1,23 @@
 (* TODO merge with src/core/Engine_kind.ml *)
```

Comment:
you can remove this comment.
Maybe you can say
(* Note that we also have Engine_kind.t, which is an alias to semgrep_output_v1.engine_type with
 * Just OSS | Pro. This might seem redundant with Engine_kind but here we are more concerned
 * about configuring the engine, whereas Engine_kind.t is more useful as metrics for the users
 *)